packages/elements/list/src/getListDeleteForward.ts

```diff
@@ -52,6 +97,34 @@ export const getListDeleteForward = (editor: SPEditor) => {
       )[0];
 
       if (!liWithSiblings) {
+        // there are no more list item in the list
+        const afterPoint = Editor.after(editor, listItem[1]);
+
+        if (afterPoint) {
+          // there is a block after it
+          const nextSiblingListRes = getListItemEntry(editor, {
+            at: afterPoint.path,
```

Comment:
Any specific reason not to use the following?

```suggestion
            at: afterPoint,
```